src/Files.App/ViewModels/Dialogs/BulkRenameDialogViewModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Windows.Storage;
+
+namespace Files.App.ViewModels.Dialogs
+{
+	public sealed class BulkRenameDialogViewModel : ObservableObject
+	{
+		private IContentPageContext context { get; } = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		// Properties
+
+		public bool IsNameValid =>
+			FilesystemHelpers.IsValidForFilename(_FileName) && !_FileName.Contains(".");
```

Comment:
Why not if it contains a dot? There should be no problem with including a dot in a file name even if it doesn't support changing the extension.

Reply:
Until we implement a preview feature for renamed items, allowing dots in filenames may cause confusion when users add file extensions unnecessarily. Although this will prevent bulk renaming of files with a dot, I believe it is a better alternative for now.

Reply:
I feel that we can just display a note saying that the dot is reflected in the filename not the extension, but if that is by design, then it is fine.

Reply:
Fair enough, I think we'll be able to make improvements with future updates.